geth/jail/jail_test.go

```diff
@@ -115,6 +116,44 @@ func (s *JailTestSuite) TestFunctionCall() {
 	require.Equal(expectedResponse, response)
 }
 
+func (s *JailTestSuite) TestJailRPCAsyncSend() {
+	require := s.Require()
+	var wg sync.WaitGroup
```

Comment:
I was sticking to the approach of `Declare -Run - Assert`, will refactor as advised.